app/javascript/packs/application.scss

```
@@ -1,5 +1,7 @@
+@import "bootstrap/scss/bootstrap";
```

```suggestion
@import "~bootstrap/scss/bootstrap";
```
If you change this, you should be able to drop the new gems 🥳